src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/PostEntity.java

```diff
@@ -2,12 +2,14 @@
 
 import com.google.common.base.Predicate;
 import com.google.common.collect.Iterables;
+import com.wikia.webdriver.common.core.WikiaWebDriver;
 import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
 import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.WebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)